src/screen/playing.rs

```diff
@@ -0,0 +1,32 @@
+use bevy::{input::common_conditions::input_just_pressed, prelude::*};
+
+use super::Screen;
+
+pub(super) fn plugin(app: &mut App) {
+    // Screen setup and teardown.
+    app.add_systems(OnEnter(Screen::Playing), enter_playing)
+        .add_systems(OnExit(Screen::Playing), exit_playing);
+
+    app.add_systems(
+        Update,
+        return_to_title_screen
+            .run_if(in_state(Screen::Playing).and_then(input_just_pressed(KeyCode::Escape))),
+    );
+}
+
+fn enter_playing(mut commands: Commands, asset_server: Res<AssetServer>) {
+    commands.spawn((
+        SpriteBundle {
+            texture: asset_server.load("ducky.png"),
+            ..Default::default()
+        },
+        StateScoped(Screen::Playing),
+    ));
+}
+
+// TODO: Reset camera transform here.
+fn exit_playing() {}
```

Comment:
What do you mean by this?

Reply:
If the camera moves around while you're playing (e.g. follow the player), which may not be universal but happens in almost all games, its transform should be reset to 0, 0 when you exit the playing state.